testHelpers/utils.ts

```diff
@@ -10,7 +10,9 @@ import {toWei, getApprovalDigest} from '../testHelpers/permitUtils';
 const BN = ethers.BigNumber.from;
 
 import {
-  ERC1155ERC721__factory,
+  //ERC1155ERC721__factory,
```

Comment:
Please remove this line.

Reply:
Removed. Will be included in next commit.